github/github-workflow.xml

```diff
@@ -0,0 +1,77 @@
+<Workflow name="Qmasters GitHub workflow for QRadar" version="1.0" xmlns="http://qradar.ibm.com/UniversalCloudRESTAPI/Workflow/V2">
+    <Parameters>
+        <Parameter name="orgName" label="Organization Name" required="false" />
+        <Parameter name="apiToken" label="API Token" required="true" secret="true" />
+        <Parameter name="phrase" label="Search Phrase" required="false"/>
+        <Parameter name="perPage" label="Per Page" required="false" default="30" />
+        <Parameter name="include" label="Include" required="false" default="web" />
+        <Parameter name="timeZone" label="Time Zone" required="false" default="UTC" />
+    </Parameters>
+
+    <Actions>
+        <!-- Clear the log source status before a new workflow run starts -->
+        <ClearStatus />
+
+        <!-- Initialize the risingValue filter for the first run. -->
```

Comment:
add a description of how long we fetch in the first time in this value <Initialize path="/bookmark" value="${time() - 3600000}" /> also, maybe can we extract it as a parameter for the user? and set it 1 hr by def.

Reply:
description added and added startTime parameter to use for the first run.